distribution/docs/src/main/resources/_contents/_apis/application-service-api-contents.adoc

```diff
@@ -0,0 +1,131 @@
+
+Applications should be defined in a `<name>=<format>` syntax where location may be empty for applications that have already been added to ${branding} or were prepackaged with the distribution.
+
+.Examples:
+[source]
+----
+# Local application:
+opendj-embedded
+
+# Application installed into a local maven repository:
+opendj-embedded=mvn:org.codice.opendj.embedded/opendj-embedded-app/1.0.1-SNAPSHOT/xml/features
+
+# Application located on the file system:
+opendj-embedded=file:/location/to/opendj-embedded-app-1.0.1-SNAPSHOT.kar
+----
+
+Applications will be started in the order they are listed in the file.
+If an application is listed, ${branding} will also attempt to install all dependencies for that application.
+
+==== Application Service Interfaces
+
+. `ApplicationService` 
+. `Application`
+. `ApplicationStatus`
+. `ApplicationNode`
+
+===== `ApplicationService` Interface
+
+The ApplicationService interface is the main class that is used to operate on applications.
+
+`getApplications`:: This method returns a set of all applications that are installed on the system. Callers can then use the Application handle to get the name and any underlying features and bundles that this application contains.
+`getApplications`:: Returns the application that has the given name.
+`startApplication`:: Starts an application, including any defined dependencies in the application.
+`stopApplication`:: Stops an application, does not include any external transitive dependencies as they may be needed by other applications.
+`addApplication`:: Adds a new application to the application list. *NOTE: This does NOT start the application.*
+`removeApplication`:: Removes an application that has the given URI.
+`isApplicationStarted`:: This method takes in an application and returns a boolean value relating whether the application is started or not. This method is generally called after retrieving a list of applications in the first method.
+`getApplicationStatus`:: This method, unlike `isApplicationStarted`, returns the full status of an application. This status contains detailed information about the health of the application and is described in the `ApplicationStatus` interface section.
+`getApplicationTree`:: Creates a hierarchy tree of application nodes that show the relationship between applications.
+`findFeature`:: Determine which application contains a certain feature.
+
+===== `Application` Interface
+
+`getName`:: Name of the application. Should be unique among applications.
+`getFeatures`:: Retrieves all of the features that this application contains regardless if they are required.
+`getBundles`:: Retrieves all of the bundles that are defined by the features and included in this application.
+
+===== `ApplicationStatus`
+
+`getApplication`:: Sends back the application that is associated with this status.
+`getState`:: Returns the application's state as defined by ApplicationState.
```

Comment:
@ricklarsen do we need to add to document ApplicationState? Not sure if there was resolution on that.